<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySQLCreatorDialogForm</class>
 <widget class="QDialog" name="MySQLCreatorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySQL Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="m_datasourceTypeTitleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>MySQL</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QGroupBox" name="m_basicOptionsGroupBox">
           <property name="title">
            <string>Server Information</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLabel" name="m_hostNameLabel">
                      <property name="text">
                       <string>Host Name/IP</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="m_hostNameLineEdit">
                      <property name="maximumSize">
                       <size>
                        <width>110</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QLabel" name="m_userNameLabel">
                      <property name="text">
                       <string>User</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="m_userNameLineEdit">
                      <property name="maximumSize">
                       <size>
                        <width>110</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <item>
                       <widget class="QLabel" name="m_portLabel">
                        <property name="text">
                         <string>Port</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="m_portLineEdit">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="inputMask">
                         <string>999999999; </string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>13</width>
                        <height>38</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QLabel" name="m_passwordLabel">
                      <property name="text">
                       <string>Password</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="m_passwordLineEdit">
                      <property name="maximumSize">
                       <size>
                        <width>110</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="echoMode">
                       <enum>QLineEdit::Password</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="m_schemaLabel">
                  <property name="text">
                   <string>Database/Schema Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="m_schemaNameComboBox">
                  <property name="maximumSize">
                   <size>
                    <width>230</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="m_charsetLabel">
                  <property name="text">
                   <string>Charset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="m_charsetComboBox">
                  <property name="maximumSize">
                   <size>
                    <width>230</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="m_basicOptionsGroupBox_2">
           <property name="title">
            <string>Creation Information</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="m_userNameLabel_3">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>New Database/Schema Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="m_newSchemaNameLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>230</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="m_advancedOptionsCheckBox">
           <property name="text">
            <string>Advanced Creation options</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="m_advancedOptionsGroupBox">
         <property name="title">
          <string>Advanced Creation Options</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_18">
                  <item>
                   <widget class="QLabel" name="m_hostNameLabel_2">
                    <property name="text">
                     <string>New Host Name/IP</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="m_newHostNameLineEdit"/>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <widget class="QLabel" name="m_userNameLabel_2">
                    <property name="text">
                     <string>New User</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="m_newUserNameLineEdit"/>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_21">
                  <item>
                   <widget class="QLabel" name="m_clientEncodingLabel_3">
                    <property name="text">
                     <string>New Collate</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="m_newCollateComboBox">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_19">
                    <item>
                     <widget class="QLabel" name="m_portLabel_2">
                      <property name="text">
                       <string>New Port</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="m_newPortLineEdit">
                      <property name="maximumSize">
                       <size>
                        <width>70</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="inputMask">
                       <string>9999999999; </string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>27</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_16">
                  <item>
                   <widget class="QLabel" name="m_passwordLabel_2">
                    <property name="text">
                     <string>New Password</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="m_newPasswordLineEdit">
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QLabel" name="m_clientEncodingLabel_2">
                    <property name="text">
                     <string>New Charset</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="m_newCharsetComboBox">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="m_createRasterMetadataCheckBox">
              <property name="text">
               <string>Create Raster Metadata Tables</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="m_createOGCMetadataCheckBox">
              <property name="text">
               <string>Create OGC Metadata Tables</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="m_helpPushButton">
           <property name="text">
            <string>&amp;Help</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="m_applyPushButton">
             <property name="text">
              <string>&amp;Apply</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_closePushButton">
             <property name="text">
              <string>&amp;Close</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_hostNameLineEdit</tabstop>
  <tabstop>m_portLineEdit</tabstop>
  <tabstop>m_userNameLineEdit</tabstop>
  <tabstop>m_passwordLineEdit</tabstop>
  <tabstop>m_schemaNameComboBox</tabstop>
  <tabstop>m_charsetComboBox</tabstop>
  <tabstop>m_newSchemaNameLineEdit</tabstop>
  <tabstop>m_advancedOptionsCheckBox</tabstop>
  <tabstop>m_applyPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
  <tabstop>m_newPasswordLineEdit</tabstop>
  <tabstop>m_newCollateComboBox</tabstop>
  <tabstop>m_newCharsetComboBox</tabstop>
  <tabstop>m_createRasterMetadataCheckBox</tabstop>
  <tabstop>m_createOGCMetadataCheckBox</tabstop>
  <tabstop>m_newHostNameLineEdit</tabstop>
  <tabstop>m_newPortLineEdit</tabstop>
  <tabstop>m_newUserNameLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
